tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/game_preferences/multiz_performance.tsx

```diff
@@ -5,6 +5,7 @@ export const multiz_performance: Feature<number> = {
   category: 'GAMEPLAY',
   description: 'How detailed multi-z is. Lower this to improve performance',
   component: createDropdownInput({
+    [-2]: 'High',
```

Comment:
The default should be the highest, swap these around and make sure it stays at -1 so that existing players get the benefit.

Reply:
I'll swap as soon as unit tests are done, thanks!

Reply:
-2 is not an option by the way

Reply:
@Mothblocks what that number is the amount of lower z levels to render before entering low spec mode. -1 is a special case